exec/plugin-commands-script-runners/src/regexp-command.ts

```diff
@@ -0,0 +1,15 @@
+export function buildRegExpFromCommand (
+  command: string
+): RegExp | null {
+  if (command.length < 3) {
+    return null
+  }
+  if (command[0] !== '/' || command.lastIndexOf('/') < 1) {
```

Comment:
You can use endsWith, no?

```suggestion
  if (command[0] !== '/' || !command.endsWith('/')) {
```

Reply:
RegExp literal expressions such as `/~~/` can also represent flagged regular expressions such as `/~~/g`. The reason why  I did not use `endsWith`  is that `endsWith` will be false if it is flagged, and the conditional branching will not work.

Of course, I fell almost all of users does not pass flags when actually specifying scripts with regular expressions, but on the other hand, it would be confusing if we do not support them...